styles/main.scss

```diff
@@ -171,36 +171,45 @@ select.search-input:disabled {
     border: 1px solid $disabled;
 }
 
-#reset-button {
+#reset-container {
```

Comment:
seems a bit strange that we are only using flexbox for the reset button within the container.  If we used flexbox to arrange the title + selects + reset button, perhaps we could center the reset button on mobile
![image](https://user-images.githubusercontent.com/66332/130343276-4f3f2e04-50ca-4992-ab29-ff1cdb3bc887.png)